sdcm/cluster.py

```diff
@@ -444,6 +444,19 @@ def init_system(self):
 
         return self._init_system
 
+    def write_msg_to_journalctl(self, msg):
```

Comment:
I understood that it is need to be written to journalctl log on node 
From task:
"when node is terminated by AWS add a message to journalctl that the node is going to be terminated."
We need only in sct log?

Reply:
the main purpose of the task that we need to get notified in the SCT log. Since we get the journalctl and you could implement some process on the machine that prints to journalctl and since we catch journalctl we get the print to the log "for free". This was the logic behind the printout to journalctl :)

Reply:
i see